independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/Beans.java

```diff
@@ -363,11 +348,13 @@ static BeanInfo createProducerField(FieldInfo producerField, BeanInfo declaringB
             alternativePriority = initAlternativePriority(producerField, alternativePriority, stereotypes, beanDeployment);
             // after all attempts, priority is still null
             if (alternativePriority == null) {
-                LOGGER.debugf(
-                        "Ignoring producer field %s - declared as an @Alternative but not selected by @Priority, @AlternativePriority or quarkus.arc.selected-alternatives",
-                        producerField);
+                String message = "Ignoring producer field %s - declared as an @Alternative but not selected by @AlternativePriority or quarkus.arc.selected-alternatives";
```

Comment:
Why remove the "`@Priority`" part? :-)

Reply:
Because `@Priority` has `@Target({TYPE,PARAMETER})` and so it cannot be used for producers...

Reply:
I see, thanks!